### PR TITLE
III-4463 Add attendance as exportable field

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added attendance as exportable field

---
Ticket: https://jira.uitdatabank.be/browse/III-4463

### CODE REVIEW

File: src/core/translations/dutch-translations.constant.js [hunk 1]
```diff
@@ -329,6 +329,7 @@ angular.module('udb.core')
       'contactPoint': 'Contactinfo',
       'status': 'Status',
       'bookingAvailability': 'Tickets & plaatsen',
+      'attendance': 'Aanwezigheidsvorm (fysiek / online)',
```
Comment (bertramakers) [inline src/core/translations/dutch-translations.constant.js:332]:
```suggestion
      'attendance': 'Aanwezigheidsvorm (fysiek / online / beide)',
```
?

Comment (bertramakers) [inline src/core/translations/dutch-translations.constant.js:332]: @corneelwille is "Aanwezigheidsvorm" the copy that you decided on? I remember you asking for feedback about how to translate it in Slack a while back and that you were going to check with marcom.

Comment (LucWollants) [reply src/core/translations/dutch-translations.constant.js:332]: Also see the proposal of adding "Beide" https://github.com/cultuurnet/udb3-angular/pull/714#discussion_r870369168

Comment (corneelwille) [reply src/core/translations/dutch-translations.constant.js:332]: Good!

**Pull request merged**